src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala

```diff
@@ -176,14 +180,16 @@ class TezosPlatformDiscoveryOperations(
   def getEntities: Future[List[Entity]] = {
     val result = for {
       entities <- caching.getEntities(networkName)
-      res <- if (entities.get._1 + cacheTTL.toMillis > now) {
-        IO.pure(entities.get._2)
-      } else {
-        for {
-          updatedEntities <- IO.fromFuture(IO(metadataOperations.runQuery(preCacheEntities)))
-          _ <- caching.putAllEntities(updatedEntities)
-        } yield updatedEntities(networkName)._2
-      }
+      res <- entities.toList.map { case CacheEntry(last, ent) =>
```

Comment:
mechanical translation

`.map(...).sequence` is always `.traverse(...)`

Reply:
true, didn't notice that, changed